### ISSUE TITLE
[Music-Triggers-1.18.2-fabric-5.4-hf] Music trigger and song registering in ShowDebugInfo, no sound. 

### ISSUE THREAD

**Issue by Ridiix** [TheComputerizer/Music-Triggers#55]
**Describe the bug**
Some songs I added are registering in the debuginfo but there is no sound. 

My current trigger document: 

```
[songsofwarostifwepullthisoff]`
    [songsofwarostifwepullthisoff.trigger]
        name = "time"
        priority = "1"
        start = "day"
        identifier = "d"
        fade_in = "500"
        fade_out = "500"
        delay = "250"

[songsofwarostsendaria]
    [songsofwarostsendaria.trigger]
        name = "time"
        identifier = "d"

[songsofwarostunderwood]
    [songsofwarostunderwood.trigger]
        name = "time"
        identifier = "d"

[songsofwarostridingsouth]
    [songsofwarostridingsouth.trigger]
        name = "time"
        identifier = "d"

[songfowarostarrivingintown]
    [songfowarostarrivingintown.trigger]
        name = "time"
        identifier = "d"

[songsofwarostcrownpeak]
    [songsofwarostcrownpeak.trigger]
        name = "time"
        identifier = "d"

[songsofwarostnotafarmer]
    [songsofwarostnotafarmer.trigger]
        name = "time"
        identifier = "d"

[songsofwarostcatchingup]
    [songsofwarostcatchingup.trigger]
        name = "time"
        identifier = "d"

[songsofwarosthogtown]
    [songsofwarosthogtown.trigger]
        name = "time"
        identifier = "d"

[nightmiquellashaligtree]
     [nightmiquellashaligtree.trigger]
        name = "time"
        priority = "2"
        start = "night"
        identifier = "n"
        fade_in = "500"
        fade_out = "500"
        delay = "250"

[nightcravesouls]
     [nightcravesouls.trigger]
        name = "time"
        identifier = "n"
        
[nightsongsofwarostnorthwind]
     [nightsongsofwarostnorthwind.trigger]
        name = "time"
        identifier = "n"

[songsofwarostwhispersunderthestars]
     [songsofwarostwhispersunderthestars.trigger]
        name = "time"
        identifier = "n"

[songsofwarosttalkoftheprimes]
     [songsofwarosttalkoftheprimes.trigger]
        name = "time"
        identifier = "n"

[songsofwarostclanless]
     [songsofwarostclanless.trigger]
        name = "time"
        identifier = "n"

[songsofwarostthrenody]
     [songsofwarostthrenody.trigger]
        name = "time"
        identifier = "n"

[daysecretbetrayal]
    [daysecretbetrayal.trigger]
	name = "time"
        identifier = "n"

[nightpremonition]
    [nightpremonition.trigger]
	name = "time"
        identifier = "n"
```
Files have all been converted to .ogg format correctly and play correctly on multiple software's. 

Not sure if I have done something wrong here, I'm pretty new to this.

Thanks!

**Comment by TheComputerizer** [comment]
The still could be converted incorrectly. If the music discs for the songs cannot play the songs themselves cannot either. Audacity usually converts correctly if this is the issue

**Comment by TheComputerizer** [comment]
Whatever the issue was is probably fixed in 6.0 now due to the new sound engine. Feel free to open another issue if you continue to have problems
